products/bash/release-ace-integration-server.sh

```diff
@@ -239,7 +247,7 @@ while [ "$numberOfMatchesForImageTag" -ne "$numberOfReplicas" ]; do
 
   echo -e "\nINFO: Total $is_release_name demo pods deployed with new image: $numberOfMatchesForImageTag"
   echo -e "\nINFO: All current $is_release_name demo pods are:\n"
-  oc get pods -n $namespace | grep $is_release_name | grep 1/1 | grep Running
+  oc get pods -n $namespace | grep $is_release_name | grep 3/3 | grep Running
```

Comment:
EEI passes `-p 1` to use 1 replica, DDD uses 3 replicas. I guess this check needs to work for both cases.

Reply:
Done